packages/workflow-core/src/lib/workflow-runner.ts

```diff
@@ -69,7 +69,7 @@ export class WorkflowRunner {
   #__subscriptions: Partial<Record<string, Array<(event: WorkflowEvent) => Promise<void>>>>;
   #__workflow: StateMachine<any, any, any>;
   #__currentState: string | undefined | symbol | number | any;
-  #__context: any;
+  context: any;
```

Comment:
Should still denote its private by convention. Also if you can check if `private` works better it will be appreciated.
```suggestion
  __context: any;
```

Reply:
I changed the `#__` because the IDE's debugger could not process and show it's value